### PR TITLE
chore(README): add telemetry information to README

### PR DESCRIPTION
**Pull request by joshuachp**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,11 +80,12 @@ period = 60
 ```
 
 #### [Astarte Message Hub](https://github.com/astarte-platform/astarte-message-hub)
+
 A central service that runs on (Linux) devices for collecting and delivering messages from N apps
 using 1 MQTT connection to Astarte.
 
-**N.B.** When using this option the Astarte Message Hub should already be installed and running
-on your system.
+**N.B.** When using this option the Astarte Message Hub should already be installed and running on
```
Comment (atsetilam) [inline README.md:87]: I would add a comma after _option,_ to improve readability.

File: README.md [hunk 2]
```diff
@@ -100,6 +102,31 @@ enabled = true
 period = 60
 ```
 
+## Telemetry
+
+Edgehog sends telemetry from interfaces defined in the
```
Comment (atsetilam) [inline README.md:107]: I might be out of context here, but wouldn't "Edgehog sends telemetry _data_ from interfaces" fit better?

**Review by atsetilam — APPROVED** [review]
Quibbling and pettifogging.

File: README.md [hunk 3]
```diff
@@ -100,6 +102,31 @@ enabled = true
...
+[edgehog-astarte-interfaces](https://github.com/edgehog-device-manager/edgehog-astarte-interfaces)
+repository. Here's how to configure some key values.
+
+### Image Version
+
+The device runtime extracts the image version from the `/etc/os-release` file. Example:
```
Comment (harlem88) [inline README.md:113]:
```suggestion
The device runtime extracts the image name and version from the `/etc/os-release` file. Example:
```

File: README.md [hunk 4]
```diff
@@ -100,6 +102,31 @@ enabled = true
...
+
+```sh
+# /etc/os-release
+IMAGE_ID="..."
```
Comment (harlem88) [inline README.md:117]: Add also the `IMAGE_VERSION`, 
```suggestion
IMAGE_ID="..."
IMAGE_VERSION="..."
```

**Review by harlem88 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (joshuachp) [reply README.md:87]: Done

File: README.md (hunk 3 above)
Comment (joshuachp) [reply README.md:113]: Done

File: README.md (hunk 4 above)
Comment (joshuachp) [reply README.md:117]: Done

**Review by harlem88 — APPROVED** [review]

**Review by harlem88 — APPROVED** [review]

**Pull request merged**